### ISSUE TITLE
[lr-policy] is lr-policy feature useful to mxnet?

### ISSUE THREAD

**Issue by qiaohaijun** [dmlc/mxnet#796]
I am a caffe user, caffe solver configure file has a lr_policy option, it can be set to "step" as a example.
so I want to know, is this feature useful or not?

**Comment by wangg12** [comment]
Same quention here. I want some policies like cyclical learning rate(CLR).

**Comment by winstywang** [comment]
see misc.py

**Comment by wangg12** [comment]
@winstywang Only FactorScheduler in misc.py?

**Comment by winstywang** [comment]
You could implement more as needed. The interface is already there :)

**Comment by qiaohaijun** [comment]
@winstywang thx for the important tip

**Comment by mli** [comment]
misc.py is deprecated due to the mismatch on the distributed setting. please have a look at 

https://github.com/dmlc/mxnet/blob/master/python/mxnet/lr_scheduler.py

this is what we used on example/image-classification.
